src/tests/functions/gif/test_gif.py

```diff
@@ -21,15 +21,16 @@ def test_gif_with_param(body):
     assert result == gifs[body["params"][0]]
 
 
-def test_gif_error():
+def test_gif_recherche():
     body = {"params": ["JamesDoe"]}
 
     request_mock = MagicMock()
     request_mock.get_json.return_value = body
 
     result = gif(request_mock)
+    url_part = "https://media.tenor.com/"
 
-    assert result == UNKNOWN_GIF
+    assert url_part in result
```

Comment:
You are not testing anything here ?

Reply:
i mean, i just make sure that the api returns a gif url like it's supposed to

Reply:
:pika:
Tu as changé le code non ? C'etait pas ca avant XD